fcrepo-jms/pom.xml

```diff
@@ -47,6 +47,10 @@
       <artifactId>javax.inject</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.modeshape</groupId>
```

Comment:
Do you see what requires `org.modeshape` in the `fcrepo-jms` module? It seems like that is the next target.